Explorer/Assets/DCL/Roads/Pools/RoadAssetsPool.cs

```diff
@@ -21,11 +40,30 @@ public RoadAssetsPool(IReadOnlyList<GameObject> roadPrefabs)
             roadAssetParent = new GameObject("ROAD_ASSET_POOL").transform;
             roadAssetPoolDictionary = new Dictionary<string, IObjectPool<Transform>>();
 
+
             foreach (GameObject gameObject in roadPrefabs)
             {
                 IObjectPool<Transform> roadAssetPool
                     = new ObjectPool<Transform>(() => Object.Instantiate(gameObject, roadAssetParent).transform,
-                        t => t.gameObject.SetActive(true), t => t.gameObject.SetActive(false), actionOnDestroy: UnityObjectUtils.SafeDestroyGameObject);
+                        t => t.gameObject.SetActive(true), t => t.gameObject.SetActive(false), actionOnDestroy: UnityObjectUtils.SafeDestroyGameObject,
+                        defaultCapacity: DEFAULT_POOL_CAPACITY);
+
+                // Long roads are very heavy assets, in terms of memory, whose pools should not pre-warmed
+                if (!gameObject.CompareTag("LongRoad"))
```

Comment:
Is there some way we can have just one instance of each `LongRoads` prewarmed? Its all that we will need in the execution

Reply:
Sure, we can do that.

Reply:
Done.